doc/source/configuration.rst

```diff
@@ -34,20 +34,32 @@ Otherwise, ``cASO`` will not be able to get the usages and will fail.
 In order to do so, we are going to setup a new role ``accounting`` a new user
 ``accounting``, adding it to each of the projects with that role::
 
-    openstack role create accounting
     openstack user create --password <password> accounting
     # For each of the projects, add the user with the accounting role
-    openstack role add --user accounting --project <project> accounting
+    openstack role add --user accounting --project <project> reader
+
+Moreover, if you wish to gather the list of projects that ``cASO`` should use directly
+from OpenStack, you need to grant the user the role reader with a system scope of
+``all``::
+
+    openstack role add --system all --user accounting reader
 
 Policy modifications
---------------------
-The accounting user needs access to Keystone so as to extract the users
-information. In this case, we can can grant the user just the rights for
-listing the users adding the appropriate rules in your policy configuration.
+------------------------
```

Comment:
this one is a bit confusing, I understand that with current OpenStack versions, what is needed is that `reader` role is added to the user that runs cASO, and no further changes are needed (so this whole Policy modifications section would be useless). Is this correct? 

If so, then I would adjust the previous section, by removing the 
```
In order to do so, we are going to setup a new role ``accounting`` a new user
``accounting``, adding it to each of the projects with that role::
```

And then moving this "Policy modifications" to something like "Using a dedicated role" or similar